### PR TITLE
Adding NPL support

### PR DESCRIPTION
**Pull request by lavanya-f5**
**Description**:  Added Support for NodePortLocal(Antrea CNI) feature with CIS.

## General Checklist
- [x] Updated Added functionality/ bug fix in release notes
- [x] Added examples for new feature
- [x] Updated the documentation
- [x] Smoke testing completed

### CODE REVIEW

File: pkg/appmanager/appManager.go [hunk 1]
```diff
@@ -3387,3 +3481,95 @@ func getIngressV1Backend(ing *netv1.Ingress) []string {
...
+		}
+	}
+	return svcList
+}
+
+func (appMgr *Manager) GetPodsForService(namespace, serviceName string) *v1.PodList {
+	svcKey := namespace + "/" + serviceName
+	crInf, ok := appMgr.getNamespaceInformer(namespace)
+	if !ok {
+		log.Errorf("Informer not found for namespace: %v", namespace)
```
Comment (sravyap135) [inline pkg/appmanager/appManager.go:3521]: missing return..

File: pkg/appmanager/appManager.go [hunk 2]
```diff
@@ -3387,3 +3481,95 @@ func getIngressV1Backend(ing *netv1.Ingress) []string {
...
+	}
+	svc, _, err := crInf.svcInformer.GetIndexer().GetByKey(svcKey)
+	if err != nil {
+		log.Infof("Error fetching service %v from the store: %v", svcKey, err)
```
Comment (sravyap135) [inline pkg/appmanager/appManager.go:3525]: error log and missing return

**Review by sravyap135 — APPROVED** [review]
LGTM

File: pkg/appmanager/appManager.go (hunk 2 above)
Comment (lavanya-f5) [reply pkg/appmanager/appManager.go:3526]: Added log and return

**Pull request merged**